myuw/templates/future_quarters.html

```diff
@@ -8,14 +8,6 @@
   <script>
     // TODO: Replace this with a object in the context and a json_script.
```

Comment:
Hi Sarthak, do you still want to replace this with a object in the context?

Reply:
not right now, this is very low priority

Reply:
sounds good.